apps/emqx_cluster_link/src/emqx_cluster_link_schema.erl

```diff
@@ -0,0 +1,79 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2024 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%--------------------------------------------------------------------
+
+-module(emqx_cluster_link_schema).
+
+-behaviour(emqx_schema_hooks).
+
+-include_lib("hocon/include/hoconsc.hrl").
+
+-export([injected_fields/0]).
+
+-export([
+    roots/0,
+    fields/1,
+    namespace/0,
+    desc/1
+]).
+
+-define(MQTT_HOST_OPTS, #{default_port => 1883}).
+
+namespace() -> "cluster_linking".
+
+roots() -> [].
+
+injected_fields() ->
+    #{cluster => fields("cluster_linking")}.
+
+fields("cluster_linking") ->
+    [
+        {links,
+            ?HOCON(?ARRAY(?R_REF("link")), #{default => [], validator => fun links_validator/1})}
```

Comment:
add a `desc` for this field and explain that there is a hidden `enable` flag which can be used to temporary disable a link while keeping the config context.